skins/cat17/src/scripts/donationForm.js

```diff
@@ -295,7 +295,16 @@ $( function () {
 				'donationFormContent.email',
 				WMDE.StateAggregation.donorTypeAndAddressAreValid
 			]
-		}
+		},
+      {
+        viewHandler: WMDE.View.createWarningBoxHandler(
```

Comment:
We probably can get away without a new view handler (warning_box.js)

      {
	  	viewHandler: WMDE.View.createElementClassSwitcher(
			$( '#street, #adress-company' ).nextAll( '.warning-text' ),
			/^([^0-9]*)$/,
			'problematic'
		),
		stateKey: 'donationFormContent.street'
      }

Reply:
Ah yes. Was that stuff added after error_box or something? I've now removed warning_box and used `createSimpleVisibilitySwitcher` instead.